.github/workflows/release-tasks.yml

```diff
@@ -84,12 +84,21 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install dependencies
-        run: sudo apt-get install -y python3-setuptools
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-setuptools python3-psutil clang cmake
```

Comment:
I think we should maybe use the `setup-cpp` action that installs clang from our apt.llvm.org instead and we can pin the version, so it doesn't change under us randomly and then we get Ninja as well.

Reply:
```
  - name: Setup Cpp
    uses: aminya/setup-cpp@v1
    with:
      compiler: llvm-16.0.6
      cmake: true
      ninja: true
```

Reply:
I've just pushed a fix for this.